app/models/inbox.rb

```diff
@@ -66,6 +66,10 @@ def web_widget?
     channel.class.name.to_s == 'Channel::WebWidget'
   end
 
+  def inbox_type
+    channel.class.name.to_s.downcase.remove('channel::', 'page', 'profile', 'sms').strip
```

Comment:
I would suggest to add a method `name` on Channel to resolve the name instead of this. This is just to ensure that the Twilio Whatsapp channel returns `Whatsapp` and not `Twilio`

```rb
def name
  medium === :whatsapp ? 'Whatsapp' : 'Twilio'
end
```